### PR TITLE
Add gender support

### PR DESCRIPTION
**Pull request by sduenas**
This PR fulfils the requirements proposed in #128.

### CODE REVIEW

File: sortinghat/api.py [hunk 1]
```diff
@@ -446,6 +448,23 @@ def edit_profile(db, uuid, **kwargs):
...
+
+            if not profile.gender:
+                gender_acc = None
+            else:
+                gender_acc = kwargs.get('gender_acc', 100)
+
+                if type(gender_acc) != int:
+                    raise WrappedValueError('gender_acc must have an integer value')
+                elif not 0 <= gender_acc <= 100:
+                    raise WrappedValueError('gender accuracy is not in range (0,100)')
```
Comment (acs) [inline sortinghat/api.py]: I would use `gender_acc` instead of `gender accuracy`. Maybe `gender_acc (gender_accuracy)`

File: sortinghat/api.py [hunk 2]
```diff
@@ -446,6 +448,23 @@ def edit_profile(db, uuid, **kwargs):
...
+
+            profile.gender_acc = gender_acc
+        elif 'gender_acc' in kwargs:
+            raise WrappedValueError('gender accuracy is set only when gender is given')
```
Comment (acs) [inline sortinghat/api.py]: The same as above

File: tests/test_api.py [hunk 3]
```diff
@@ -50,7 +50,10 @@
 COUNTRY_CODE_INVALID_ERROR = "country code must be a 2 length alpha string - %(code)s given"
 ENROLLMENT_PERIOD_INVALID_ERROR = "cannot be greater than "
 ENROLLMENT_PERIOD_OUT_OF_BOUNDS_ERROR = "%(type)s %(date)s is out of bounds"
-NOT_FOUND_ERROR =  "%(entity)s not found in the registry"
+NOT_FOUND_ERROR = "%(entity)s not found in the registry"
+GENDER_ACC_INVALID_ERROR = "gender accuracy is set only when gender is given"
+GENDER_ACC_INVALID_TYPE_ERROR = "gender_acc must have an integer value"
+GENDER_ACC_INVALID_RANGE_ERROR = "gender accuracy is not in range (0,100"
```
Comment (acs) [inline tests/test_api.py]: (0,100) ?

Comment (sduenas) [reply tests/test_api.py]: You're probably right and 0 doesn't make any sense.

**Comment by acs** [comment]
LGTM the changes after the review!

**Pull request merged**